returnn/torch/engine.py

```diff
@@ -1663,27 +1627,3 @@ def _get_total_grad_norm(model: torch.nn.Module, p: float) -> float:
             p=p,
         ).item()
     )
-
-
-def _get_num_seqs_last_seq_idx(
```

Comment:
I completely removed this function, because we only access `num_seqs` once and just for printing it to the log, so I don't think the assertions add enough value to warrant the additional verbosity. Is that ok with you or do you prefer to keep this around?

Reply:
It's ok.